Paper: `kernel`->`extern`, `length`->`duration` (#204)
Qiskit/openqasm: source/language/classical.rst

Classical instructions
======================

We envision two levels of classical control: simple, low-level instructions embedded as
part of a quantum circuit and high-level external functions which perform more complex

> classical computations. The low-level functions are fast, apply control flow and allow basic

Reviewer: ```suggestion
classical computations. The low-level functions allow basic
```
I think we can simplify this further. In particular, OpenQASM doesn't have branching _instructions_, but it does have branching _syntax_.

Author: Addressed in https://github.com/Qiskit/openqasm/pull/204/commits/40df87657528bc9c5aa65355cc79175c01763fc7.

computations on lower-level parallel control processors. These instructions are likely
to have known durations and many such instructions might be executed
within the qubit coherence time. The external, or ``extern``, functions execute
complex blocks of classical code that may be neither fast nor guaranteed to return. In order
to connect with classical compilation infrastucture, ``extern`` functions are defined outside of

> quantum circuits. The compiler toolchain is expected to link ``extern`` functions when building an

Reviewer: ```suggestion
OpenQASM. The compiler toolchain is expected to link ``extern`` functions when building an
```

Author: Addressed in https://github.com/Qiskit/openqasm/pull/204/commits/a18f2cb9de95408601a0a1abf9c3ae1caccd5497

> executable. This strategy allows the programmer to use existing libraries with porting them into

Reviewer: ```suggestion
executable. This strategy allows the programmer to use existing libraries without porting them into
```

Author: Addressed in https://github.com/Qiskit/openqasm/pull/204/commits/a18f2cb9de95408601a0a1abf9c3ae1caccd5497.
